src/main/resources/instructorEmailFragment-instructorCopyPreamble.html

```diff
@@ -1,5 +1,13 @@
 <p>
   The email below has been sent to students of course: [${courseId}] ${courseName}.
   <br><br>
+  Kindly note that this email simply serves as a preview of how the email will appear to the
+  students, and the link is not the actual link that the students will receive.
+  As such, please do not forward this email to students.
```

Comment:
Maybe using some kind of formatting here to emphasize `do not forward`, could be bolding/colors/etc

Reply:
Made the text `do not forward` bold and red.